### ISSUE TITLE
Eleventy won't include liquid file

### ISSUE THREAD

**Issue by Naseki** [11ty/eleventy#950]
It throws the following error:
```
AssertionError: cannot include with empty filename
```

Code is the following:
```
---
layout: layout.liquid
pageTitle: Welcome to my blog
---

{% assign allposts = collections.posts %}
{% include postlist %}
```
Here's what the folder hierarchy looks like (code is inside index.liquid). As you can see, postlist.liquid is present and should be in the right folder. It's also able to find layout.liquid just fine.

![Capture](https://user-images.githubusercontent.com/30054237/74848789-4dfe3200-5338-11ea-8ba3-14ecb2c7bf38.PNG)

.eleventyignore contains a single folder that isn't used anywhere.

I've tried putting quotes around the name (dynamicPartials set to true) which results to `TypeError: undefined filter: 1` instead.

Is there something I've done wrong, or have I encountered a bug by any chance?

**Comment by pdehaan** [comment]
> {% include postlist %}

Does it work if you use `{% include postlist.liquid %}`?

**Comment by Naseki** [comment]
@pdehaan Forgot to mention I tried that as well, and it throws the same AssertionError.

**Comment by pdehaan** [comment]
Interesting. I wasn't able to reproduce it with a similar repo: https://github.com/pdehaan/11ty-950 (macOS, w/ eleventy v0.10.0).

**Comment by Naseki** [comment]
I found where the problem lies and could reproduce this.

Simply adding this to .eleventy.js results to the error:
```
module.exports = function(eleventyConfig) {
    eleventyConfig.setLiquidOptions({
      dynamicPartials: true,
      strict_filters: true
    });
  };
```
Perhaps I have a poor understanding of what this actually does.

**Comment by pdehaan** [comment]
Interesting, I tried that in https://github.com/pdehaan/11ty-950/blob/master/index.liquid, and got the error, but then adding quotes around the partial name fixed the error for me:

```liquid
---
layout: layout.liquid
pageTitle: Welcome to my blog
---

{% assign allposts = collections.posts %}
{% include 'postlist' %}
```

Which also matches the documentation at https://www.11ty.dev/docs/languages/liquid/#quoted-include-paths

**Comment by Naseki** [comment]
By doing that, as stated in the original post, I got the error `TypeError: undefined filter: 1` instead. However, I just found out that this error is caused by something unrelated to all of this, so I'll consider this fixed.

Let's say this whole problem was a mix of me not knowing that setting dynamicPartials to true makes it a _requirement_ to quote your includes, while the other issue was lying there all along.
